src/ComposerRequireChecker/DependencyGuesser/GuessFromLoadedExtensions.php

```diff
@@ -2,15 +2,24 @@
 
 namespace ComposerRequireChecker\DependencyGuesser;
 
+use ComposerRequireChecker\Cli\Options;
 use ComposerRequireChecker\DefinedSymbolsLocator\LocateDefinedSymbolsFromExtensions;
 
 class GuessFromLoadedExtensions implements GuesserInterface
 {
     private $loadedExtensions;
 
-    public function __construct()
+    /**
+     * @var array|null
```

Comment:
`array` of what? Can you please annotate the type, such as `string[]` or `ExtensionName[]`?